style.css

```diff
@@ -1,3 +1,123 @@
+* {
+  box-sizing: content-box;
+}
+header {
+  background-color: rgb(14, 41, 66);
+  padding: 15px;
+  color: white;
+  margin-top: auto;
+  border-radius: 6px;
+}
+form {
+  float: right;
+  clear: both;
+  font-size: 17px;
+}
+
+label {
+  float: right;
+  padding-right: 8px;
+}
+
+select {
+  padding: 10px;
+  font-size: 12px;
+  height: 15px;
+}
+
+h1 {
+  font: outline;
+  font-size: 60px;
+  padding-left: 450px;
+}
+#search-bar {
+  text-align: left;
+  font-size: medium;
+  border: 1px solid;
+  padding: 10px;
+}
+
+#select-episode {
+  width: 220px;
+}
+
+#episode-container {
+  color: black;
+
+  border-radius: 6px;
+}
 #root {
-  color: red;
+  display: flex;
+  flex-flow: row wrap;
+  background-color: rgb(189, 216, 216);
+}
+
+.card {
+  display: inline-flex;
+  flex-basis: auto;
+  flex-flow: column;
+  flex-grow: 1;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+  padding: 1rem;
+  border: 2px #f0e68c solid;
+  border-radius: 8px;
+  width: 18rem;
+  min-height: 100vh;
```

Comment:
The cards are currently very tall, which makes it hard to fit many on the page. I would probably remove the `min-height`.

You could set a `max-height` and an `overflow: hidden` on the text to prevent it from spilling over.